api/src/main/java/com/datatorrent/api/ControlTupleEnabledSink.java

```diff
@@ -18,7 +18,7 @@
  */
 package com.datatorrent.api;
 
-import org.apache.apex.api.UserDefinedControlTuple;
+import org.apache.apex.api.ControlTuple;
```

Comment:
I don't think we should club platform and operator developers into one category. It is apt to club operator developers with application developers as users of the platform. The fact that BW, EW and CHK are implemented as special tuples by the platform is unknown to the users nor should they have to care about it and by extension they shouldn't have to call their control tuples by some other distinguishing name such as user defined control tuples.